.github/workflows/upgrade-e2e.yml

```diff
@@ -0,0 +1,41 @@
+---
+on:
+  pull_request:
+  push:
+    branches:
+      - master
+    tags:
```

Comment:
I think we don't want to run tests like this on tags, per https://github.com/submariner-io/submariner/commit/b3f568d90458897ae0d5881008a6064812888c25.